### ISSUE TITLE
Interactive Hydra tutorial with Katacoda

### ISSUE THREAD

**Issue by omry** [facebookresearch/hydra#696]
An interactive tutorial with [Katakoda](https://www.katacoda.com/) would be awesome.

As an example, see the [Kubernetes tutorial](https://kubernetes.io/docs/tutorials/kubernetes-basics/create-cluster/cluster-interactive/)

**Comment by pranavdurai10** [comment]
Hey @omry, is this still open for contribution?

**Comment by omry** [comment]
Yes, I think interactive versions of the tutorials would be a great addition to Hydra.

**Comment by pranavdurai10** [comment]
I would like to contribute. Can you explain the nature of work? @omry

**Comment by omry** [comment]
At a high level, create interactive versions of the Hydra tutorials in Katacoda.
Take a look at the Kubernetes tutorial linked above to get an idea about how Katakoda can be used.

This is creative work, I can't really describe it in detail.

**Comment by omry** [comment]
Closing as part of an automated issue cleanup sweep.

This request appears to be obsolete because it was specific to Katacoda, whose public tutorial platform is no longer available. If this is still relevant, or if I closed this by mistake, please comment here with updated details and we can reopen it.
